engine/src/main/java/org/terasology/rendering/nui/animation/SwipeMenuAnimationSystem.java

```diff
@@ -181,10 +181,14 @@ public void update(float delta) {
             // avoid skipping over fast animations on heavy load
             animDelta = 0.1f;
         }
+        if(animDelta == 0.0f) {
+            // in case time is paused
```

Comment:
You could expand on this comment a bit with our discoveries about how the game being paused stops time and results in zeroed out delta values, so to make sure the animations still run we "fake" that time is still running at a set rate